Dockerfile

```diff
@@ -579,6 +580,7 @@ RUN --mount=type=bind,target=. \
   set -e
   target=$([ "$DOCKER_STATIC" = "1" ] && echo "binary" || echo "dynbinary")
   xx-go --wrap
+  export PKG_CONFIG=$(xx-go env PKG_CONFIG)
```

Comment:
This still feels a little hacky to me; I feel like prefixing `PKG_CONFIG=` to just `make.sh` might be a bit more clear as to intent.